tests/composite_pk/models/tenant.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+
+class Tenant(models.Model):
+    name = models.CharField(max_length=10, default="", blank=True)
+
+
+class Token(models.Model):
+    id = models.SmallIntegerField()
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE, related_name="tokens")
+    secret = models.CharField(max_length=10, default="", blank=True)
+
+    class Meta:
+        primary_key = ("id",)
+
+
+class BaseModel(models.Model):
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE)
+    id = models.SmallIntegerField(unique=True)
+
+    class Meta:
+        abstract = True
+        primary_key = ("tenant_id", "id")
+
+
+class User(BaseModel):
+    email = models.EmailField(default="", blank=True)
+
+
+class Comment(models.Model):
+    tenant = models.ForeignKey(
+        Tenant,
+        on_delete=models.CASCADE,
+        related_name="comments",
+    )
+    id = models.SmallIntegerField(unique=True, db_column="comment_id")
+    user_id = models.SmallIntegerField()
+    user = models.ForeignObject(
+        User,
+        on_delete=models.CASCADE,
+        from_fields=("tenant_id", "user_id"),
+        to_fields=("tenant_id", "id"),
+        related_name="comments",
+    )
+    text = models.TextField(default="", blank=True)
+
+    class Meta:
+        primary_key = ("tenant_id", "id")
```

Comment:
Do we support using real field names for `ForeignKey`? It would be nice if this can be:

```suggestion
        primary_key = ("tenant", "id")
```

Reply:
Yes, I recently pushed the changes to support this

Reply:
Cool! I saw you'd changed `channel_id` to `id` here, but was curious if there was a reason not to change `tenant_id` too.